src/main/java/org/opentripplanner/graph_builder/GraphBuilder.java

```diff
@@ -340,7 +360,7 @@ public static InputFileType forFile(File file) {
             if (name.endsWith(".osm")) return OSM;
             if (name.endsWith(".osm.xml")) return OSM;
             if (name.endsWith(".tif") || name.endsWith(".tiff")) return DEM; // Digital elevation model (elevation raster)
-            if (name.equals("Graph.obj")) return GRAPH;
+            if (name.endsWith(".obj")) return BASEGRAPH;
```

Comment:
I have two concerns with this-
- Is behavior undefined if there are multiple ".obj" files in a directory? I would think you only load one base graph.
- This changes default behavior for if there is a pre-existing Graph.obj file in a directory. Right now, behavior is that Graph.obj is overwritten with a new graph. With this change, you'd load the graph in and create new data structures on top of it. I'm concerned this would be unexpected to a lot of OTP users.

Reply:
I added a new parameter to specify the graph name to be loaded as a base graph. This should make functionality identical for current users. I have tested building from base graph and normal graph building several times and it seems to work as intended.